frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -249,6 +249,10 @@ const ManageHostsPage = ({
     queryParams?.software_id !== undefined
       ? parseInt(queryParams?.software_id, 10)
       : undefined;
+  const status =
+    queryParams?.status !== "" && isAcceptableStatus(queryParams?.status)
```

Comment:
This checks that the status is acceptable before rendering

Reply:
what do you think about removing `queryParams?.status !== ""`? it will make the ternary easier to read, and seems like `isAcceptableStatus` will still return `false` if the string is empty.